src/renderer/components/layout/page-layout.tsx

```diff
@@ -80,11 +82,13 @@ export class PageLayout extends React.Component<PageLayoutProps> {
             <RecursiveTreeView data={navigation}/>
           </nav>
         )}
-        <div className="content-wrapper" id="ScrollSpyRoot">
-          <div className={cssNames("content", contentClass, contentGaps && "flex column gaps")}>
-            {children}
+        <CacheProvider value={createCache({ key: "page-layout" })}>
```

Comment:
Why do we wrap `CacheProvider` only in `PageLayout` and `MainLayout`?
Isn't it should be inside `app.tsx` and `lens-app.tsx`? Otherwise how you can be sure where `react-select` actually used internally in app?

Reply:
Moved `CacheProvider` into `app.tsx` & `lens-app.tsx` covering more components at once.